import.go

```diff
@@ -227,23 +227,28 @@ func acquireUrl(c types.StackerConfig, storage types.Storage, i string, cache st
 
 	// It's just a path, let's copy it to .stacker.
 	if url.Scheme == "" {
-		return importFile(i, cache, expectedHash)
+		return importFile(importPath, cache, expectedHash)
 	} else if url.Scheme == "http" || url.Scheme == "https" {
 		// otherwise, we need to download it
+
+		if hashRequired && expectedHash == "" {
+			return "", errors.Errorf("Remote import needs a hash in yaml for path: %s", importPath)
```

Comment:
can we do this validation in NewStackerFile() so that it fails faster vs. halfway through the build?

Reply:
Done. Thanks!